specs/jsonschema-core.md

```diff
@@ -2461,9 +2461,7 @@ Bowman, Gowry Sankar, Donald Pipowitch, Dave Finlay, Denis Laxalde, Phil
 Sturgeon, Shawn Silverman, and Karen Etheridge for their submissions and patches
 to the document.
 
-## [Appendix] Change Log[^19]
-
-[^19]: This section to be removed before leaving Internet-Draft status.
+## [Appendix] Change Log
```

Comment:
We should either remove the section or leave the note.

Reply:
Oh yeha, I forgot to write a not about that change. This came up because the linter got confused about having a footnote on a heading. I didn't think the note was necessary (especially because it's talking about Internet-Drafts), so I just removed it. I'm in favor of removing the whole section if you're ok with that. Otherwise we can keep the footnote by moving it into the body.

Reply:
I'm okay with either.  I was going to remove these sections anyway, but in a separate PR where that's the focus.

Reply:
Ok. Let's do it as a separate PR. I'll restore the footnote, but move it so the linter isn't confused.

Reply:
So, it turns out that it's not that the linter was confused about being in a heading. The rule is that footnotes always go at the bottom of the document and for some reason it's not catching all the other cases where the footnote is next to where it's declared.